components/prism-php.js

```diff
@@ -114,6 +126,25 @@
 	// The different types of PHP strings "replace" the C-like standard string
 	delete Prism.languages.php['string'];
 
+	Prism.languages.insertBefore('php', 'class-name', {
+		'class-static-call': {
+			pattern: /[a-zA-Z_][\w\\]+(?=\:\:)/,
+			greedy: !0,
+			alias: 'class-name'
+		},
+		'class-type-hint': {
+			pattern: /[a-zA-Z_][\w\\]+(?=\s*\$)/,
+			greedy: !0,
+			alias: 'class-name'
+		},
+		'class-return-type': {
+			pattern: /(\)\s*\:\s*\?*\s*)[a-zA-Z_][\w\\]+/,
```

Comment:
```suggestion
			pattern: /(\)\s*:\s*\?*\s*)[a-zA-Z_][\w\\]*/,
```

Reply:
fixed

Reply:
here should be `\??` instead of `\?*`.

Reply:
yes, i'll fix it in https://github.com/PrismJS/prism/pull/2591